src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -4364,43 +4364,43 @@ s32 func_80838FB8(PlayState* play, Player* this) {
  * Note: grottos and normal fairy fountains use `ENTR_RETURN_GROTTO`
  */
 s16 sReturnEntranceGroupData[] = {
-    // ENTR_RETURN_DAIYOUSEI_IZUMI
-    /*  0 */ ENTR_DEATH_MOUNTAIN_TRAIL_4,  // DMT from Magic Fairy Fountain
-    /*  1 */ ENTR_DEATH_MOUNTAIN_CRATER_3, // DMC from Double Defense Fairy Fountain
-    /*  2 */ ENTR_HYRULE_CASTLE_2,         // Hyrule Castle from Dins Fire Fairy Fountain
+    // ENTR_RETURN_GREAT_FAIRYS_FOUNTAIN_MAGIC
+    /*  0 */ ENTR_DEATH_MOUNTAIN_TRAIL_4,  // from Magic Fairy Fountain
+    /*  1 */ ENTR_DEATH_MOUNTAIN_CRATER_3, // from Double Magic Fairy Fountain
+    /*  2 */ ENTR_HYRULE_CASTLE_2,         // from Double Defense Fairy Fountain (as adult)
 
     // ENTR_RETURN_2
-    /*  3 */ ENTR_KAKARIKO_VILLAGE_9, // Kakariko from Potion Shop
-    /*  4 */ ENTR_MARKET_DAY_5,       // Market (child day) from Potion Shop
+    /*  3 */ ENTR_KAKARIKO_VILLAGE_9, // from Potion Shop
+    /*  4 */ ENTR_MARKET_DAY_5,       // from Potion Shop
 
-    // ENTR_RETURN_SHOP1
-    /*  5 */ ENTR_KAKARIKO_VILLAGE_3, // Kakariko from Bazaar
-    /*  6 */ ENTR_MARKET_DAY_6,       // Market (child day) from Bazaar
+    // ENTR_RETURN_BAZAAR
+    /*  5 */ ENTR_KAKARIKO_VILLAGE_3,
+    /*  6 */ ENTR_MARKET_DAY_6,
```

Comment:
Since `ENTR_RETURN_BAZAAR` is named, could also name `ENTR_RETURN_2` -> `ENTR_RETURN_POTION_SHOP`? But not a big deal if it's out of scope

Reply:
It is not the same potion shop though, probably at one point it was the same scene but then the kakariko version got the passage to the back